pytorch_blade/tests/disc/ops/test_binary_ops.py

```diff
@@ -84,18 +84,18 @@ def _test_binary_ops(self, binary_ops_func, test_int=True):
             out, res = self._test_cvt_to_disc(binary_ops_func, test_data)
             self._check_type(out, res)
 
-    def _test_func(self, torch_func):
+    def _test_func(self, torch_func, test_int=True):
```

Comment:
#391 has merged, please add some shape annotation to test dynamic shape test case.

Reply:
done